[package] ftw/3.3.10: Recipe should request an older CMake

### Description

fftw cannot be compiled with the latest CMake as support for CMake < 3.5 has been dropped

### Package and Environment Details

* Package Name/Version: **fftw/3.3.10**
* Operating System+version: **macOS**
* Compiler+version: **Apple clang 15.0**
* Conan version: **conan 2.20.1**
* Python version: **Python 3.11.9**


### Conan profile

Profile host:
[settings]
arch=x86_64
build_type=Debug
compiler=apple-clang
compiler.cppstd=20
compiler.libcxx=libc++
compiler.version=15
os=Macos
[options]
libtype=static

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=apple-clang
compiler.cppstd=gnu17
compiler.libcxx=libc++
compiler.version=15
os=Macos


### Steps to reproduce

conan install --requires fftw/3.3.10

### Logs

https://github.com/mmomtchev/ImageMagick/actions/runs/17711457972/job/50330685364

<details><summary>Click to expand log</summary>

```
 conan profile detect || echo ok
  conan install . -of build --build=missing -o libtype=static --settings=build_type=Debug --settings compiler.cppstd=20
  conan cache clean
  echo 'CONAN_TOOLCHAIN=-DCMAKE_TOOLCHAIN_FILE="/Users/runner/work/ImageMagick/ImageMagick/build/conan_toolchain.cmake"' >> $GITHUB_ENV
  shell: /bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    PKG_CONFIG_PATH: /Users/runner/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
    Python_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    Python2_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    Python3_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
detect_api: Found apple-clang 15.0
detect_api: apple-clang>=13, using the major as version

Detected profile:
WARN: This profile is a guess of your environment, please check it.
WARN: Defaulted to cppstd='gnu17' for apple-clang.
WARN: The output of this command is not guaranteed to be stable and can change in future Conan versions.
WARN: Use your own profile files for stability.
Saving detected profile to /Users/runner/.conan2/profiles/default
[settings]
arch=x86_64
build_type=Release
compiler=apple-clang
compiler.cppstd=gnu17
compiler.libcxx=libc++
compiler.version=15
os=Macos

WARN: legacy: Unscoped option definition is ambiguous.
Use '&:libtype=static' to refer to the current package.
Use '*:libtype=static' or other pattern if the intent was to apply to dependencies

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Debug
compiler=apple-clang
compiler.cppstd=20
compiler.libcxx=libc++
compiler.version=15
os=Macos
[options]
libtype=static

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=apple-clang
compiler.cppstd=gnu17
compiler.libcxx=libc++
compiler.version=15
os=Macos


======== Computing dependency graph ========
libffi/3.4.4: Not found in local cache, looking in remotes...
libffi/3.4.4: Checking remote: conancenter
Connecting to remote 'conancenter' anonymously
libffi/3.4.4: Downloaded recipe revision 33feb967b22d92ea0becce63bf9d5490
fontconfig/2.14.2: Not found in local cache, looking in remotes...
fontconfig/2.14.2: Checking remote: conancenter
fontconfig/2.14.2: Downloaded recipe revision 8f382f75366114d5f7d2c5714ce3c438
freetype/2.13.2: Not found in local cache, looking in remotes...
freetype/2.13.2: Checking remote: conancenter
freetype/2.13.2: Downloaded recipe revision 18656f7a6d52256a930f1cbd79f1509d
libpng/1.6.42: Not found in local cache, looking in remotes...
libpng/1.6.42: Checking remote: conancenter
libpng/1.6.42: Downloaded recipe revision df487b07ca898d97bf3fad8988639c67
zlib/1.2.13: Not found in local cache, looking in remotes...
zlib/1.2.13: Checking remote: conancenter
zlib/1.2.13: Downloaded recipe revision 9df41c65e2c2b6ef47633dc32e0b699a
bzip2/1.0.8: Not found in local cache, looking in remotes...
bzip2/1.0.8: Checking remote: conancenter
bzip2/1.0.8: Downloaded recipe revision 00b4a4658791c1f06914e087f0e792f5
brotli/1.1.0: Not found in local cache, looking in remotes...
brotli/1.1.0: Checking remote: conancenter
brotli/1.1.0: Downloaded recipe revision 406ce8f1c997f4ef7852fa01ff85ef9f
meson/1.8.2: Not found in local cache, looking in remotes...
meson/1.8.2: Checking remote: conancenter
meson/1.8.2: Downloaded recipe revision 996f3893995475dbe96a0dce6503b0c8
ninja/1.13.1: Not found in local cache, looking in remotes...
ninja/1.13.1: Checking remote: conancenter
ninja/1.13.1: Downloaded recipe revision 294f8721dbcde145674f7ba44994700e
pkgconf/2.2.0: Not found in local cache, looking in remotes...
pkgconf/2.2.0: Checking remote: conancenter
pkgconf/2.2.0: Downloaded recipe revision 4ac315b50ef734072b00ff3aacbf52bf
expat/2.6.0: Not found in local cache, looking in remotes...
expat/2.6.0: Checking remote: conancenter
expat/2.6.0: Downloaded recipe revision 2f92fb3cf504e9731aba6d20dcd641bf
gperf/3.1: Not found in local cache, looking in remotes...
gperf/3.1: Checking remote: conancenter
gperf/3.1: Downloaded recipe revision a7afdf8f7cccdc2dcd4d962370c33d4f
pkgconf/2.1.0: Not found in local cache, looking in remotes...
pkgconf/2.1.0: Checking remote: conancenter
pkgconf/2.1.0: Downloaded recipe revision 21f96520faf7660b99f872e956d2ac13
fribidi/1.0.12: Not found in local cache, looking in remotes...
fribidi/1.0.12: Checking remote: conancenter
fribidi/1.0.12: Downloaded recipe revision 789f9577b1c288fc0221ca8052289cdf
meson/1.2.1: Not found in local cache, looking in remotes...
meson/1.2.1: Checking remote: conancenter
meson/1.2.1: Downloaded recipe revision f2b0c7763308df8e33172744dace8845
glib/2.78.1: Not found in local cache, looking in remotes...
glib/2.78.1: Checking remote: conancenter
glib/2.78.1: Downloaded recipe revision d80e99a84b177829df9565c1c44f3124
pcre2/10.42: Not found in local cache, looking in remotes...
pcre2/10.42: Checking remote: conancenter
pcre2/10.42: Downloaded recipe revision 9a35f5089feb875ec61a38eca364ce77
libelf/0.8.13: Not found in local cache, looking in remotes...
libelf/0.8.13: Checking remote: conancenter
libelf/0.8.13: Downloaded recipe revision ba59bbc89757ed62cfd7690a73bf81be
autoconf/2.71: Not found in local cache, looking in remotes...
autoconf/2.71: Checking remote: conancenter
autoconf/2.71: Downloaded recipe revision 51077f068e61700d65bb05541ea1e4b0
m4/1.4.19: Not found in local cache, looking in remotes...
m4/1.4.19: Checking remote: conancenter
m4/1.4.19: Downloaded recipe revision b38ced39a01e31fef5435bc634461fd2
gnu-config/cci.20210814: Not found in local cache, looking in remotes...
gnu-config/cci.20210814: Checking remote: conancenter
gnu-config/cci.20210814: Downloaded recipe revision 69fde734e1a46fd1655b4af46ab40945
libgettext/0.22: Not found in local cache, looking in remotes...
libgettext/0.22: Checking remote: conancenter
libgettext/0.22: Downloaded recipe revision 766c603ca03bfbcad907369de5368c29
libiconv/1.17: Not found in local cache, looking in remotes...
libiconv/1.17: Checking remote: conancenter
libiconv/1.17: Downloaded recipe revision 1e65319e945f2d31941a9d28cc13c058
pkgconf/2.0.3: Not found in local cache, looking in remotes...
pkgconf/2.0.3: Checking remote: conancenter
pkgconf/2.0.3: Downloaded recipe revision c7ba7fd1d7d22534ee070ae60ed79604
harfbuzz/8.3.0: Not found in local cache, looking in remotes...
harfbuzz/8.3.0: Checking remote: conancenter
harfbuzz/8.3.0: Downloaded recipe revision 0ed431896d25d1a0b84be38fc6e0107a
gettext/0.22.5: Not found in local cache, looking in remotes...
gettext/0.22.5: Checking remote: conancenter
gettext/0.22.5: Downloaded recipe revision 4705a1582f4a611eadb15d0417427993
automake/1.16.5: Not found in local cache, looking in remotes...
automake/1.16.5: Checking remote: conancenter
automake/1.16.5: Downloaded recipe revision b91b7c384c3deaa9d535be02da14d04f
lzma_sdk/9.20: Not found in local cache, looking in remotes...
lzma_sdk/9.20: Checking remote: conancenter
lzma_sdk/9.20: Downloaded recipe revision f1fe20056422603432563a07b25f2ee3
zstd/1.5.5: Not found in local cache, looking in remotes...
zstd/1.5.5: Checking remote: conancenter
zstd/1.5.5: Downloaded recipe revision 8400bcba21d1b495dca55edf3f065246
libzip/1.9.2: Not found in local cache, looking in remotes...
libzip/1.9.2: Checking remote: conancenter
libzip/1.9.2: Downloaded recipe revision 586100044d7434d72167b965ed4109e5
xz_utils/5.4.5: Not found in local cache, looking in remotes...
xz_utils/5.4.5: Checking remote: conancenter
xz_utils/5.4.5: Downloaded recipe revision b885d1d79c9d30cff3803f7f551dbe66
openssl/3.5.2: Not found in local cache, looking in remotes...
openssl/3.5.2: Checking remote: conancenter
openssl/3.5.2: Downloaded recipe revision 0c5a5e15ae569f45dff57adcf1770cf7
fftw/3.3.10: Not found in local cache, looking in remotes...
fftw/3.3.10: Checking remote: conancenter
fftw/3.3.10: Downloaded recipe revision ae4a2efdec8d27e398260587d0ff2457
lcms/2.14: Not found in local cache, looking in remotes...
lcms/2.14: Checking remote: conancenter
lcms/2.14: Downloaded recipe revision 8a8c02ab8c0631e06d705c38b171f053
libxml2/2.10.4: Not found in local cache, looking in remotes...
libxml2/2.10.4: Checking remote: conancenter
libxml2/2.10.4: Downloaded recipe revision cff0dafab700a5aa0fd4dfdab6bbe2d9
libaom-av1/3.6.0: Not found in local cache, looking in remotes...
libaom-av1/3.6.0: Checking remote: conancenter
libaom-av1/3.6.0: Downloaded recipe revision db22f37c8c3cb56d23ac5faccda7cc75
libde265/1.0.12: Not found in local cache, looking in remotes...
libde265/1.0.12: Checking remote: conancenter
libde265/1.0.12: Downloaded recipe revision 6e7cec8c7e3b1c9bfb6761dfc4d1dbb8
libheif/1.19.5: Not found in local cache, looking in remotes...
libheif/1.19.5: Checking remote: conancenter
libheif/1.19.5: Downloaded recipe revision c5e36f9b0e764acd8fad54f1d6e9a62d
cmake/3.31.8: Not found in local cache, looking in remotes...
cmake/3.31.8: Checking remote: conancenter
cmake/3.31.8: Downloaded recipe revision dde3bde00bb843687e55aea5afa0e220
jbig/20160605: Not found in local cache, looking in remotes...
jbig/20160605: Checking remote: conancenter
jbig/20160605: Downloaded recipe revision 2d29fa02aacd76902e0d2cbbc24631ef
openexr/3.3.2: Not found in local cache, looking in remotes...
openexr/3.3.2: Checking remote: conancenter
openexr/3.3.2: Downloaded recipe revision f852dad6c283ee366bd0e14a3ae570cb
imath/3.2.1: Not found in local cache, looking in remotes...
imath/3.2.1: Checking remote: conancenter
imath/3.2.1: Downloaded recipe revision 47a001e9196b8d377c839c4725a99772
libdeflate/1.23: Not found in local cache, looking in remotes...
libdeflate/1.23: Checking remote: conancenter
libdeflate/1.23: Downloaded recipe revision 4994bea7cf7e93789da161fac8e26a53
libwebp/1.3.2: Not found in local cache, looking in remotes...
libwebp/1.3.2: Checking remote: conancenter
libwebp/1.3.2: Downloaded recipe revision 9c32dd81887041c939c97990d04d14c8
libjpeg-turbo/3.0.2: Not found in local cache, looking in remotes...
libjpeg-turbo/3.0.2: Checking remote: conancenter
libjpeg-turbo/3.0.2: Downloaded recipe revision d1f0fd9c5c6eb40289a2427e486480e3
nasm/2.15.05: Not found in local cache, looking in remotes...
nasm/2.15.05: Checking remote: conancenter
nasm/2.15.05: Downloaded recipe revision f7d6a6f3b14f583e001d9ae84e8cd152
jasper/4.2.0: Not found in local cache, looking in remotes...
jasper/4.2.0: Checking remote: conancenter
jasper/4.2.0: Downloaded recipe revision be96e122b0898c3b6ba70453ce78f01f
cmake/4.1.1: Not found in local cache, looking in remotes...
cmake/4.1.1: Checking remote: conancenter
cmake/4.1.1: Downloaded recipe revision 23ee27eac0192593228fe8ced4680627
libtiff/4.6.0: Not found in local cache, looking in remotes...
libtiff/4.6.0: Checking remote: conancenter
libtiff/4.6.0: Downloaded recipe revision 1c497ad376872d80a0ad7cf05dc422ae
libraw/0.21.2: Not found in local cache, looking in remotes...
libraw/0.21.2: Checking remote: conancenter
libraw/0.21.2: Downloaded recipe revision b170823a725a8cd8ae7abd6caf7ee7cc
openjpeg/2.5.0: Not found in local cache, looking in remotes...
openjpeg/2.5.0: Checking remote: conancenter
openjpeg/2.5.0: Downloaded recipe revision 2886244f2492121aa7c6c493a814db06
cairo/1.17.8: Not found in local cache, looking in remotes...
cairo/1.17.8: Checking remote: conancenter
cairo/1.17.8: Downloaded recipe revision 6985730ddd64ceafde4921c0c0a6075c
pixman/0.42.2: Not found in local cache, looking in remotes...
pixman/0.42.2: Checking remote: conancenter
pixman/0.42.2: Downloaded recipe revision eaaf09ff3f04bce88309fbe2716ade58
lzo/2.10: Not found in local cache, looking in remotes...
lzo/2.10: Checking remote: conancenter
lzo/2.10: Downloaded recipe revision 5725914235423c771cb1c6b607109b45
meson/1.4.0: Not found in local cache, looking in remotes...
meson/1.4.0: Checking remote: conancenter
meson/1.4.0: Downloaded recipe revision 2262941cc8fbb0099dd0c196ca2a6c01
pango/1.54.0: Not found in local cache, looking in remotes...
pango/1.54.0: Checking remote: conancenter
pango/1.54.0: Downloaded recipe revision 02eb15e8cec727142b8dadfa32d77c81
libjxl/0.10.3: Not found in local cache, looking in remotes...
libjxl/0.10.3: Checking remote: conancenter
libjxl/0.10.3: Downloaded recipe revision 30458f71d08926cf73c7a35189235921
highway/1.1.0: Not found in local cache, looking in remotes...
highway/1.1.0: Checking remote: conancenter
highway/1.1.0: Downloaded recipe revision 6672c75ba81fae074e51e48e6a15c69b
jemalloc/5.3.0: Not found in local cache, looking in remotes...
jemalloc/5.3.0: Checking remote: conancenter
jemalloc/5.3.0: Downloaded recipe revision e951da9cf599e956cebc117880d2d9f8
Graph root
    conanfile.py: /Users/runner/work/ImageMagick/ImageMagick/conanfile.py
Requirements
    brotli/1.1.0#406ce8f1c997f4ef7852fa01ff85ef9f - Downloaded (conancenter)
    bzip2/1.0.8#00b4a4658791c1f06914e087f0e792f5 - Downloaded (conancenter)
    cairo/1.17.8#6985730ddd64ceafde4921c0c0a6075c - Downloaded (conancenter)
    expat/2.6.0#2f92fb3cf504e9731aba6d20dcd641bf - Downloaded (conancenter)
    fftw/3.3.10#ae4a2efdec8d27e398260587d0ff2457 - Downloaded (conancenter)
    fontconfig/2.14.2#8f382f75366114d5f7d2c5714ce3c438 - Downloaded (conancenter)
    freetype/2.13.2#18656f7a6d52256a930f1cbd79f1509d - Downloaded (conancenter)
    fribidi/1.0.12#789f9577b1c288fc0221ca8052289cdf - Downloaded (conancenter)
    glib/2.78.1#d80e99a84b177829df9565c1c44f3124 - Downloaded (conancenter)
    harfbuzz/8.3.0#0ed431896d25d1a0b84be38fc6e0107a - Downloaded (conancenter)
    highway/1.1.0#6672c75ba81fae074e51e48e6a15c69b - Downloaded (conancenter)
    imath/3.2.1#47a001e9196b8d377c839c4725a99772 - Downloaded (conancenter)
    jasper/4.2.0#be96e122b0898c3b6ba70453ce78f01f - Downloaded (conancenter)
    jbig/20160605#2d29fa02aacd76902e0d2cbbc24631ef - Downloaded (conancenter)
    jemalloc/5.3.0#e951da9cf599e956cebc117880d2d9f8 - Downloaded (conancenter)
    lcms/2.14#8a8c02ab8c0631e06d705c38b171f053 - Downloaded (conancenter)
    libaom-av1/3.6.0#db22f37c8c3cb56d23ac5faccda7cc75 - Downloaded (conancenter)
    libde265/1.0.12#6e7cec8c7e3b1c9bfb6761dfc4d1dbb8 - Downloaded (conancenter)
    libdeflate/1.23#4994bea7cf7e93789da161fac8e26a53 - Downloaded (conancenter)
    libelf/0.8.13#ba59bbc89757ed62cfd7690a73bf81be - Downloaded (conancenter)
    libffi/3.4.4#33feb967b22d92ea0becce63bf9d5490 - Downloaded (conancenter)
    libgettext/0.22#766c603ca03bfbcad907369de5368c29 - Downloaded (conancenter)
    libheif/1.19.5#c5e36f9b0e764acd8fad54f1d6e9a62d - Downloaded (conancenter)
    libiconv/1.17#1e65319e945f2d31941a9d28cc13c058 - Downloaded (conancenter)
    libjpeg-turbo/3.0.2#d1f0fd9c5c6eb40289a2427e486480e3 - Downloaded (conancenter)
    libjxl/0.10.3#30458f71d08926cf73c7a35189235921 - Downloaded (conancenter)
    libpng/1.6.42#df487b07ca898d97bf3fad8988639c67 - Downloaded (conancenter)
    libraw/0.21.2#b170823a725a8cd8ae7abd6caf7ee7cc - Downloaded (conancenter)
    libtiff/4.6.0#1c497ad376872d80a0ad7cf05dc422ae - Downloaded (conancenter)
    libwebp/1.3.2#9c32dd81887041c939c97990d04d14c8 - Downloaded (conancenter)
    libxml2/2.10.4#cff0dafab700a5aa0fd4dfdab6bbe2d9 - Downloaded (conancenter)
    libzip/1.9.2#586100044d7434d72167b965ed4109e5 - Downloaded (conancenter)
    lzma_sdk/9.20#f1fe20056422603432563a07b25f2ee3 - Downloaded (conancenter)
    lzo/2.10#5725914235423c771cb1c6b607109b45 - Downloaded (conancenter)
    openexr/3.3.2#f852dad6c283ee366bd0e14a3ae570cb - Downloaded (conancenter)
    openjpeg/2.5.0#2886244f2492121aa7c6c493a814db06 - Downloaded (conancenter)
    openssl/3.5.2#0c5a5e15ae569f45dff57adcf1770cf7 - Downloaded (conancenter)
    pango/1.54.0#02eb15e8cec727142b8dadfa32d77c81 - Downloaded (conancenter)
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77 - Downloaded (conancenter)
    pixman/0.42.2#eaaf09ff3f04bce88309fbe2716ade58 - Downloaded (conancenter)
    xz_utils/5.4.5#b885d1d79c9d30cff3803f7f551dbe66 - Downloaded (conancenter)
    zlib/1.2.13#9df41c65e2c2b6ef47633dc32e0b699a - Downloaded (conancenter)
    zstd/1.5.5#8400bcba21d1b495dca55edf3f065246 - Downloaded (conancenter)
Build requirements
    autoconf/2.71#51077f068e61700d65bb05541ea1e4b0 - Downloaded (conancenter)
    automake/1.16.5#b91b7c384c3deaa9d535be02da14d04f - Downloaded (conancenter)
    bzip2/1.0.8#00b4a4658791c1f06914e087f0e792f5 - Downloaded (conancenter)
    cmake/3.31.8#dde3bde00bb843687e55aea5afa0e220 - Downloaded (conancenter)
    cmake/4.1.1#23ee27eac0192593228fe8ced4680627 - Downloaded (conancenter)
    gettext/0.22.5#4705a1582f4a611eadb15d0417427993 - Downloaded (conancenter)
    glib/2.78.1#d80e99a84b177829df9565c1c44f3124 - Downloaded (conancenter)
    gnu-config/cci.20210814#69fde734e1a46fd1655b4af46ab40945 - Downloaded (conancenter)
    gperf/3.1#a7afdf8f7cccdc2dcd4d962370c33d4f - Downloaded (conancenter)
    libelf/0.8.13#ba59bbc89757ed62cfd7690a73bf81be - Downloaded (conancenter)
    libffi/3.4.4#33feb967b22d92ea0becce63bf9d5490 - Downloaded (conancenter)
    libgettext/0.22#766c603ca03bfbcad907369de5368c29 - Downloaded (conancenter)
    libiconv/1.17#1e65319e945f2d31941a9d28cc13c058 - Downloaded (conancenter)
    m4/1.4.19#b38ced39a01e31fef5435bc634461fd2 - Downloaded (conancenter)
    meson/1.2.1#f2b0c7763308df8e33172744dace8845 - Downloaded (conancenter)
    meson/1.4.0#2262941cc8fbb0099dd0c196ca2a6c01 - Downloaded (conancenter)
    meson/1.8.2#996f3893995475dbe96a0dce6503b0c8 - Downloaded (conancenter)
    nasm/2.15.05#f7d6a6f3b14f583e001d9ae84e8cd152 - Downloaded (conancenter)
    ninja/1.13.1#294f8721dbcde145674f7ba44994700e - Downloaded (conancenter)
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77 - Downloaded (conancenter)
    pkgconf/2.0.3#c7ba7fd1d7d22534ee070ae60ed79604 - Downloaded (conancenter)
    pkgconf/2.1.0#21f96520faf7660b99f872e956d2ac13 - Downloaded (conancenter)
    pkgconf/2.2.0#4ac315b50ef734072b00ff3aacbf52bf - Downloaded (conancenter)
    zlib/1.2.13#9df41c65e2c2b6ef47633dc32e0b699a - Downloaded (conancenter)
Resolved version ranges
    cmake/[>=3.16 <4]: cmake/3.31.8
    cmake/[>=3.18]: cmake/4.1.1
    cmake/[>=3.19 <4]: cmake/3.31.8
    cmake/[>=3.20]: cmake/4.1.1
    imath/[>=3.1.9 <4]: imath/3.2.1
    libdeflate/[>=1.19 <2]: libdeflate/1.23
    meson/[>=1.2.2 <2]: meson/1.8.2
    meson/[>=1.2.3 <2]: meson/1.8.2
    meson/[>=1.3.2 <2]: meson/1.8.2
    meson/[>=1.4.0 <2]: meson/1.8.2
    ninja/[>=1.10.2 <2]: ninja/1.13.1
    openssl/[>=1.1 <4]: openssl/3.5.2
    pkgconf/[>=2.1.0 <3]: pkgconf/2.2.0
    pkgconf/[>=2.2 <3]: pkgconf/2.2.0
    zlib/[>=1.2.11 <2]: zlib/1.2.13
Overrides
    libffi/3.4.4: [None, 'libffi/3.4.4']
    freetype/2.13.2#18656f7a6d52256a930f1cbd79f1509d: [None, 'freetype/2.13.2']
    bzip2/1.0.8#00b4a4658791c1f06914e087f0e792f5: [None, 'bzip2/1.0.8']
    libde265/1.0.12: [None, 'libde265/1.0.12']
    expat/[>=2.6.2 <3]: ['expat/2.6.0']
    libpng/[>=1.6 <2]: ['libpng/1.6.42']
    zlib/[>=1.2.10 <2]: ['zlib/1.2.13']
    zlib/[>=1.2.11 <2]: ['zlib/1.2.13']
    glib/2.78.3: ['glib/2.78.1']
    xz_utils/[>=5.4.5 <6]: ['xz_utils/5.4.5']
    zstd/[~1.5]: ['zstd/1.5.5']
    libjpeg-turbo/[>=3.0.2 <4]: ['libjpeg-turbo/3.0.2']
    libjpeg-turbo/3.0.0: ['libjpeg-turbo/3.0.2#d1f0fd9c5c6eb40289a2427e486480e3']
    lcms/2.16: ['lcms/2.14']
    jasper/4.0.0: ['jasper/4.2.0']
    pixman/0.43.4: ['pixman/0.42.2']
    fontconfig/2.15.0: ['fontconfig/2.14.2']
    cairo/1.18.0: ['cairo/1.17.8']
    fribidi/1.0.13: ['fribidi/1.0.12']
    harfbuzz/[>=8.3.0]: ['harfbuzz/8.3.0']
Deprecated
    libelf/0.8.13: elfutils
WARN: risk: There are deprecated packages in the graph

======== Computing necessary packages ========
highway/1.1.0: Main binary package '0bd2bd24b9f344bc044e8bf5c9252c3b32a5d615' missing
highway/1.1.0: Checking 9 compatible configurations
highway/1.1.0: Compatible configurations not found in cache, checking servers
highway/1.1.0: '72aba308f8fd0053bdeeeedcdea2a4cb554438d4': compiler.cppstd=11
highway/1.1.0: '6096cd8e790a59ab5d8d59eb7c27018c38ad09a7': compiler.cppstd=gnu11
highway/1.1.0: '6603f724d9d86efbf9b811eec57a44fae159a5a2': compiler.cppstd=14
highway/1.1.0: 'a65c5fab359b72a691c7ffec75da226e7ba83365': compiler.cppstd=gnu14
highway/1.1.0: 'b569f8c00f61fca98f6c40406fde78a72560677d': compiler.cppstd=17
highway/1.1.0: '8251220e0b2d3774ebe587393410538b13300f32': compiler.cppstd=gnu17
highway/1.1.0: 'ac7efac47600950123d2290692ac3b6e219bee8a': compiler.cppstd=gnu20
highway/1.1.0: 'c17cb4073d57938db308fff1e5e53f9208793057': compiler.cppstd=23
highway/1.1.0: 'd6bc6f63f44eca4375c7d0399742d024365b9ef3': compiler.cppstd=gnu23
imath/3.2.1: Main binary package '980764f884ab4b766acef8a028ac8256c9a1f242' missing
imath/3.2.1: Checking 9 compatible configurations
imath/3.2.1: Compatible configurations not found in cache, checking servers
imath/3.2.1: '38d0f54bd43fe1514f9fb990fa2d5f9bfa9f1e09': compiler.cppstd=11
imath/3.2.1: '23447f0acd9c39e6dc8c3acc6a56034d16e43ecf': compiler.cppstd=gnu11
imath/3.2.1: 'f9c0c8774f2552b44ce759b6ec1615acff0d2057': compiler.cppstd=14
imath/3.2.1: '810fa46aae8a6366c69be876a1c1529ca8a70d06': compiler.cppstd=gnu14
imath/3.2.1: '95742ba1b605960352ffce78f082df5934cef044': compiler.cppstd=17
imath/3.2.1: '3f00f61b0d6a07e433051b43ab2b0e44442e2531': compiler.cppstd=gnu17
imath/3.2.1: 'bc788656b5f3206151263bcac70eb0b8478b99ba': compiler.cppstd=gnu20
imath/3.2.1: 'ba4c994dd6a339a0d219f4d7913b7790bbe68fb3': compiler.cppstd=23
imath/3.2.1: 'd6ef6b49acfa717ea41f946040939d77aa3da2dd': compiler.cppstd=gnu23
libde265/1.0.12: Main binary package '2413b12b40e9c6089ca5863355c6f06e79de194c' missing
libde265/1.0.12: Checking 9 compatible configurations
libde265/1.0.12: Compatible configurations not found in cache, checking servers
libde265/1.0.12: '9a6d940fbe545c01ae9711c0ee7ff081fdb09816': compiler.cppstd=11
libde265/1.0.12: '6644e3af710e944dc844f3fa3af4dc521e78aae1': compiler.cppstd=gnu11
libde265/1.0.12: '4c5496ee3cd3964b718c4abb061a82a387031204': compiler.cppstd=14
libde265/1.0.12: 'c78c543a9e59577c289b7a8660a8c900f8850b37': compiler.cppstd=gnu14
libde265/1.0.12: '2c40fcc7d0ef985d9774e315c581e29c7a2ea387': compiler.cppstd=17
libde265/1.0.12: 'de5cb6fa9744d5b11b652a396a0d2bdc0914b4ba': compiler.cppstd=gnu17
libde265/1.0.12: '9fe633bf53c62825e22a5f2ea3b21fbe20c4ed85': compiler.cppstd=gnu20
libde265/1.0.12: '39f1cd934b7eef6892cffd49368645cb91ef3131': compiler.cppstd=23
libde265/1.0.12: '800424adce1efd557c7d75a49e67102a91ec0bab': compiler.cppstd=gnu23
libheif/1.19.5: Main binary package 'd3580e5510a01c6522a89714f93085fda133bfb9' missing
libheif/1.19.5: Checking 3 compatible configurations
libheif/1.19.5: Compatible configurations not found in cache, checking servers
libheif/1.19.5: '79421c77a7bb6ecf9a34ce163dd5bc9a18fc796a': compiler.cppstd=gnu20
libheif/1.19.5: '48587dd9887da6b3fb63508d7a1f74e54a0c2fc0': compiler.cppstd=23
libheif/1.19.5: 'e12d5756b057007e9bc23d02a0f95fb7142846aa': compiler.cppstd=gnu23
openexr/3.3.2: Main binary package 'de5da0646e8b7834229fa470beec7d02236aa085' missing
openexr/3.3.2: Checking 9 compatible configurations
openexr/3.3.2: Compatible configurations not found in cache, checking servers
openexr/3.3.2: '83acf9643335949120d75416ff7b1c3f2756ee70': compiler.cppstd=11
openexr/3.3.2: 'f31404eb1b62698bf48bf528b04f476c7b7ad6e8': compiler.cppstd=gnu11
openexr/3.3.2: '8c0ac35955bb1176ce32071e3f671e1f0c83574d': compiler.cppstd=14
openexr/3.3.2: '3ff1bad004506c797e16e13c4bf610afc6cc69e9': compiler.cppstd=gnu14
openexr/3.3.2: 'f978ded5b87d195c3fab3795d9932a82175d347f': compiler.cppstd=17
openexr/3.3.2: 'b994c398fc0f24f371bc9ac334bcb2b90648bf72': compiler.cppstd=gnu17
openexr/3.3.2: '83e2be412a657e31b4e632ab646b2a0ae85be7a1': compiler.cppstd=gnu20
openexr/3.3.2: 'db1905f00ab14c2652749df7b16e5880d22cb4ab': compiler.cppstd=23
openexr/3.3.2: 'cb60aeeaaea518af0cfd9d45b578e213d953df73': compiler.cppstd=gnu23
libtiff/4.6.0: Main binary package '7947c14267600de76495a92da35454d64bf38ddb' missing
libtiff/4.6.0: Checking 11 compatible configurations
libtiff/4.6.0: Compatible configurations not found in cache, checking servers
libtiff/4.6.0: 'b7aaf25623531b933df21fa6a0956e7c082ebbc9': compiler.cppstd=98
libtiff/4.6.0: '430c3d66bfe3e7e5277bc435b1e7a3b52861eb32': compiler.cppstd=gnu98
libtiff/4.6.0: '4d31b7d77edc3b62ae2051b3c17d47604b64a4e9': compiler.cppstd=11
libtiff/4.6.0: '8a3925d40bc46157f2c0c3c3906cbf377665c153': compiler.cppstd=gnu11
libtiff/4.6.0: '8fd250c6ceb267d07539c607fa5033aeebd11f49': compiler.cppstd=14
libtiff/4.6.0: '5bddf4797928a42c994fcca6e818325e33d8a58f': compiler.cppstd=gnu14
libtiff/4.6.0: 'befc0c2a1ccb99a1e7d6f9d320a9c90fbd90f731': compiler.cppstd=17
libtiff/4.6.0: 'dc7b17f4824a3515568feb45552b2e50d616d926': compiler.cppstd=gnu17
libtiff/4.6.0: 'f64841a2957c01193a889d39d3c6316fbfca7858': compiler.cppstd=gnu20
libtiff/4.6.0: '9c603a6e5b01c9ea67c76f00c3c54412569d7ee1': compiler.cppstd=23
libtiff/4.6.0: '39d35d006da91ac5ba1baf2359bb956e74e15d73': compiler.cppstd=gnu23
jemalloc/5.3.0: Main binary package '0c356688215ad8ac05320e35bbf699ce48f42fc5' missing
jemalloc/5.3.0: Checking 11 compatible configurations
jemalloc/5.3.0: Compatible configurations not found in cache, checking servers
jemalloc/5.3.0: '2974decdc82424a13f221862036ac18f71b10415': compiler.cppstd=98
jemalloc/5.3.0: '6b23057fbe208dab8c7772dc26f0826fd341b592': compiler.cppstd=gnu98
jemalloc/5.3.0: '7bf600ab83bb0cacc663bd971e92878c7bda3af7': compiler.cppstd=11
jemalloc/5.3.0: 'b21299eaa6ebb421143a9645fbdd7939b0cb9532': compiler.cppstd=gnu11
jemalloc/5.3.0: '16b323fa01658a0a40af89c6e100ece45f162191': compiler.cppstd=14
jemalloc/5.3.0: '56b39fe8923903b12a4c0065d1c629fa388e722d': compiler.cppstd=gnu14
jemalloc/5.3.0: 'eefb84ceec06352eae0f73f56c491a18f9eb67e7': compiler.cppstd=17
jemalloc/5.3.0: '5765c1dfef56467b95c090c6a3aaf84d34165564': compiler.cppstd=gnu17
jemalloc/5.3.0: 'e411649f15898c8575e471503c829c5d49c0711a': compiler.cppstd=gnu20
jemalloc/5.3.0: 'bcd3bb9170b9b226dcd9cfc7cdd2b68b82858598': compiler.cppstd=23
jemalloc/5.3.0: 'af42a72c6c4cb87fe39df7c180d7f0b9aacc5e66': compiler.cppstd=gnu23
libjxl/0.10.3: Main binary package 'e5380e7b99f546d104d65c7e1ac143569fdbb041' missing
libjxl/0.10.3: Checking 9 compatible configurations
libjxl/0.10.3: Compatible configurations not found in cache, checking servers
libjxl/0.10.3: 'ef41336cbfbd43accb01233dbfc106739ad0fb6d': compiler.cppstd=11
libjxl/0.10.3: 'f47b88c4da53de2032c4ad2dbe2463903710e8c8': compiler.cppstd=gnu11
libjxl/0.10.3: '98db50099aa4830f8a14b7e3547e9c0aee239c6b': compiler.cppstd=14
libjxl/0.10.3: '6c25bc0e904e63c027faaf2618d17db370eebd01': compiler.cppstd=gnu14
libjxl/0.10.3: 'd91329ef609a139d8fab170e992badc16e0ae274': compiler.cppstd=17
libjxl/0.10.3: '4b44fecfc6374f7fd028049420f816954384ae4e': compiler.cppstd=gnu17
libjxl/0.10.3: 'a6ca09e32f5eb1623b7b5b767821c33dde1ca58d': compiler.cppstd=gnu20
libjxl/0.10.3: '857bbeb6c778c31d9e8f1a572fe796727a3b9b0a': compiler.cppstd=23
libjxl/0.10.3: '68d7bf3699cd3d793af5738fb17820cbc61e170f': compiler.cppstd=gnu23
libraw/0.21.2: Main binary package 'bebcc3d412c66cfc8f25aa39b885efd996f561b6' missing
libraw/0.21.2: Checking 9 compatible configurations
libraw/0.21.2: Compatible configurations not found in cache, checking servers
libraw/0.21.2: '99153c986367b9abdbf4173541e7ae85498d36fd': compiler.cppstd=11
libraw/0.21.2: '4242a7414b9ce02d4336229a716e593da4fc81ec': compiler.cppstd=gnu11
libraw/0.21.2: '7c44d3efe3b92a3464db3f634eb0ce073cec4548': compiler.cppstd=14
libraw/0.21.2: '60712f7e70cd3ff3de62f56fe1f6f2cef498a868': compiler.cppstd=gnu14
libraw/0.21.2: 'e01fb7f7b84525c38cd93af53539e7e2b3cbd7ff': compiler.cppstd=17
libraw/0.21.2: 'a2fe9ca4984f19f96276fe94667ee9faacfc8f44': compiler.cppstd=gnu17
libraw/0.21.2: 'b9b71d5f5119d2d28faa79098de57f4755418779': compiler.cppstd=gnu20
libraw/0.21.2: '9816170f62700888ab56c6bab86d4e6c38aca129': compiler.cppstd=23
libraw/0.21.2: '3d02e349b6c1f82f67af4eef58ab1018abb37232': compiler.cppstd=gnu23
harfbuzz/8.3.0: Main binary package 'b166e89f88f284d56c4004d80219dd8a32abb628' missing
harfbuzz/8.3.0: Checking 11 compatible configurations
harfbuzz/8.3.0: Compatible configurations not found in cache, checking servers
harfbuzz/8.3.0: '8e7fb22f675fe50fd68b6cdeaa0605ada25375c4': compiler.cppstd=98
harfbuzz/8.3.0: 'c2d2c5dd5f423d17afbb65edf3158d44233125a0': compiler.cppstd=gnu98
harfbuzz/8.3.0: 'dea5f4417481a6b3e4c9c60212255760d4280e36': compiler.cppstd=11
harfbuzz/8.3.0: '9d62f26038a479782d5b626057ec06f55c06fb9b': compiler.cppstd=gnu11
harfbuzz/8.3.0: 'c2fc0ac33edca3714d6aea7798f3ef9060f13fc1': compiler.cppstd=14
harfbuzz/8.3.0: 'f36590f422f2b3df82bf60f0a547709a1b6838d4': compiler.cppstd=gnu14
harfbuzz/8.3.0: 'e304721e459a137222d823ab6f94bbbc8ab33133': compiler.cppstd=17
harfbuzz/8.3.0: '1f22a4b9026996e47645caee3e403eaec6c6e7fa': compiler.cppstd=gnu17
harfbuzz/8.3.0: 'f2b95dacad1e9a3700363c371bc5f8d7d9a5f607': compiler.cppstd=gnu20
harfbuzz/8.3.0: '2cb5b251d72ad56a0f0221760fb6bd793197a507': compiler.cppstd=23
harfbuzz/8.3.0: 'f9fa40159c6f27380b2af434b0fb66c4094adb80': compiler.cppstd=gnu23
Requirements
    brotli/1.1.0#406ce8f1c997f4ef7852fa01ff85ef9f:6c12c2f001b732562a43abd8e56f588ef1bb35df - Build
    bzip2/1.0.8#00b4a4658791c1f06914e087f0e792f5:4b147c5d0a31064bcb7488673106461a60c3ebb4 - Build
    cairo/1.17.8#6985730ddd64ceafde4921c0c0a6075c:71309686985a305ff4a687e877b5bd78c66ddbd4 - Build
    expat/2.6.0#2f92fb3cf504e9731aba6d20dcd641bf:ff2775f96a14b15e259080082b262d8ba648c7dc - Build
    fftw/3.3.10#ae4a2efdec8d27e398260587d0ff2457:4dc90630c36eda1f80f6d3bc636c419a05107dc1 - Build
    fontconfig/2.14.2#8f382f75366114d5f7d2c5714ce3c438:1c7c3cfa332b2eecc247587aac37141936e80bf3 - Build
    freetype/2.13.2#18656f7a6d52256a930f1cbd79f1509d:c0db26a71317b8e873f8855634fad4ef0f80ec8c - Build
    fribidi/1.0.12#789f9577b1c288fc0221ca8052289cdf:9112abb1783f27738ce58de354b01bd5cd183f44 - Build
    glib/2.78.1#d80e99a84b177829df9565c1c44f3124:a282490caa1c22b16f52e65393c36179a3db5630 - Build
    harfbuzz/8.3.0#0ed431896d25d1a0b84be38fc6e0107a:b166e89f88f284d56c4004d80219dd8a32abb628 - Build
    highway/1.1.0#6672c75ba81fae074e51e48e6a15c69b:0bd2bd24b9f344bc044e8bf5c9252c3b32a5d615 - Build
    imath/3.2.1#47a001e9196b8d377c839c4725a99772:980764f884ab4b766acef8a028ac8256c9a1f242 - Build
    jasper/4.2.0#be96e122b0898c3b6ba70453ce78f01f:d86ee574ef89d90cea1e584745a7a91cc61032dd - Build
    jbig/20160605#2d29fa02aacd76902e0d2cbbc24631ef:75d71da4c8cb3435e33779e156aae40ded602ff1 - Build
    jemalloc/5.3.0#e951da9cf599e956cebc117880d2d9f8:0c356688215ad8ac05320e35bbf699ce48f42fc5 - Build
    lcms/2.14#8a8c02ab8c0631e06d705c38b171f053:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    libaom-av1/3.6.0#db22f37c8c3cb56d23ac5faccda7cc75:3a69898cf88c9dfc25f8542d879ca0b52cbfa0fc - Build
    libde265/1.0.12#6e7cec8c7e3b1c9bfb6761dfc4d1dbb8:2413b12b40e9c6089ca5863355c6f06e79de194c - Build
    libdeflate/1.23#4994bea7cf7e93789da161fac8e26a53:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    libelf/0.8.13#ba59bbc89757ed62cfd7690a73bf81be:9b517ffebbfa52473a5e51f0c85ea2b7ec880a65 - Build
    libffi/3.4.4#33feb967b22d92ea0becce63bf9d5490:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    libgettext/0.22#766c603ca03bfbcad907369de5368c29:4f281a3b9ec6d44a64f14f0a8284e98652f7f53b - Build
    libheif/1.19.5#c5e36f9b0e764acd8fad54f1d6e9a62d:d3580e5510a01c6522a89714f93085fda133bfb9 - Build
    libiconv/1.17#1e65319e945f2d31941a9d28cc13c058:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    libjpeg-turbo/3.0.2#d1f0fd9c5c6eb40289a2427e486480e3:682adc6822f3ff2faf37657b19e466327e027320 - Build
    libjxl/0.10.3#30458f71d08926cf73c7a35189235921:e5380e7b99f546d104d65c7e1ac143569fdbb041 - Build
    libpng/1.6.42#df487b07ca898d97bf3fad8988639c67:2c89a27abd4bb0f8707b0ef87b93d77673e34e3f - Build
    libraw/0.21.2#b170823a725a8cd8ae7abd6caf7ee7cc:bebcc3d412c66cfc8f25aa39b885efd996f561b6 - Build
    libtiff/4.6.0#1c497ad376872d80a0ad7cf05dc422ae:7947c14267600de76495a92da35454d64bf38ddb - Build
    libwebp/1.3.2#9c32dd81887041c939c97990d04d14c8:c38f86d212be866cf7539fec458d6e1f7387b52a - Build
    libxml2/2.10.4#cff0dafab700a5aa0fd4dfdab6bbe2d9:345468fe543944ec9e5005e3511f9d165fac74ea - Build
    libzip/1.9.2#586100044d7434d72167b965ed4109e5:a8a5124cc64343f4226eb94d4c85a9d20883926b - Build
    lzma_sdk/9.20#f1fe20056422603432563a07b25f2ee3:82339cc4d6db7990c1830d274cd12e7c91ab18a1#36bd52c4dd7a7f4b470cac43f25bd621 - Download (conancenter)
    lzo/2.10#5725914235423c771cb1c6b607109b45:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    openexr/3.3.2#f852dad6c283ee366bd0e14a3ae570cb:de5da0646e8b7834229fa470beec7d02236aa085 - Build
    openjpeg/2.5.0#2886244f2492121aa7c6c493a814db06:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    openssl/3.5.2#0c5a5e15ae569f45dff57adcf1770cf7:4d6d55d80b1c2bf3f4deafb9eee0354e79524728 - Build
    pango/1.54.0#02eb15e8cec727142b8dadfa32d77c81:caf78676e84e773df17edf2b9a8eaffafd7bbf37 - Build
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77:5069803e509aad95a9da07a79a37fd2481b26f0e - Build
    pixman/0.42.2#eaaf09ff3f04bce88309fbe2716ade58:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    xz_utils/5.4.5#b885d1d79c9d30cff3803f7f551dbe66:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    zlib/1.2.13#9df41c65e2c2b6ef47633dc32e0b699a:58a582c7d3df6c4571084dad49613caab6de03ff - Build
    zstd/1.5.5#8400bcba21d1b495dca55edf3f065246:bca5a81e60066d057ea78c36de87c9e76236a82c - Build
Build requirements
    autoconf/2.71#51077f068e61700d65bb05541ea1e4b0:9e5323c65b94ae38c3c733fe12637776db0119a5#c693b1ceb6f260246f50440ff7544168 - Download (conancenter)
    automake/1.16.5#b91b7c384c3deaa9d535be02da14d04f:9e5323c65b94ae38c3c733fe12637776db0119a5#41880d59cb9b7157938fd9ee94827e51 - Download (conancenter)
    bzip2/1.0.8#00b4a4658791c1f06914e087f0e792f5:e7988bbc42945ade4ac9e0a9c78b773ecd1e91e6 - Build
    cmake/3.31.8#dde3bde00bb843687e55aea5afa0e220:9e5323c65b94ae38c3c733fe12637776db0119a5#dc85dedb7de60a1e60e0d9b131020248 - Download (conancenter)
    cmake/4.1.1#23ee27eac0192593228fe8ced4680627:9e5323c65b94ae38c3c733fe12637776db0119a5#5cc7773db11469b6c7f5db81b75593c1 - Download (conancenter)
    gettext/0.22.5#4705a1582f4a611eadb15d0417427993:4c8552cb132f62cdd4e25f34e2ce685735082ab3 - Build
    glib/2.78.1#d80e99a84b177829df9565c1c44f3124:fa1528c71b8cc351cde66e713ebc92f5270d5e5a - Build
    gnu-config/cci.20210814#69fde734e1a46fd1655b4af46ab40945:da39a3ee5e6b4b0d3255bfef95601890afd80709#22618e30bd9e326eb95e824dc90cc860 - Download (conancenter)
    gperf/3.1#a7afdf8f7cccdc2dcd4d962370c33d4f:9ac8640923e5284645f8852ef8ba335654f4020e#858dd07257747c43e443e77c5db10e5a - Download (conancenter)
    libelf/0.8.13#ba59bbc89757ed62cfd7690a73bf81be:c350cd5ac462711443ac4bfdb500d901707d084b - Build
    libffi/3.4.4#33feb967b22d92ea0becce63bf9d5490:be7ccd6109b8a8f9da81fd00ee143a1f5bbd5bbf - Build
    libgettext/0.22#766c603ca03bfbcad907369de5368c29:640f82f1944e4edde92de4ca8a09b6d0463ce647 - Build
    libiconv/1.17#1e65319e945f2d31941a9d28cc13c058:be7ccd6109b8a8f9da81fd00ee143a1f5bbd5bbf - Build
    m4/1.4.19#b38ced39a01e31fef5435bc634461fd2:9ac8640923e5284645f8852ef8ba335654f4020e#0b930e2019f5dfbf0c5e4ed8b8be9bf1 - Download (conancenter)
    meson/1.2.1#f2b0c7763308df8e33172744dace8845:da39a3ee5e6b4b0d3255bfef95601890afd80709#f7ef1db356d1746b23003b7924eb2128 - Download (conancenter)
    meson/1.4.0#2262941cc8fbb0099dd0c196ca2a6c01:da39a3ee5e6b4b0d3255bfef95601890afd80709#91b870cdcf4edb1a302a2ef7a0514791 - Download (conancenter)
    meson/1.8.2#996f3893995475dbe96a0dce6503b0c8:da39a3ee5e6b4b0d3255bfef95601890afd80709#4bc04da65e30aac060f4c7aedf23be78 - Download (conancenter)
    nasm/2.15.05#f7d6a6f3b14f583e001d9ae84e8cd152:9ac8640923e5284645f8852ef8ba335654f4020e#5b06c4e96fdf7a4baba8ad949a2ed29f - Download (conancenter)
    ninja/1.13.1#294f8721dbcde145674f7ba44994700e:9ac8640923e5284645f8852ef8ba335654f4020e#f211295e2983ea2b0e62fc97e0093251 - Download (conancenter)
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77:d3eb0018eb60950d8a2c576d903edcb7c8f00336 - Build
    pkgconf/2.0.3#c7ba7fd1d7d22534ee070ae60ed79604:8b5cf1367c351fd3c60ef4e28c1a50612761b13e#a3e961a648771db40b115af998e14ac4 - Download (conancenter)
    pkgconf/2.1.0#21f96520faf7660b99f872e956d2ac13:8b5cf1367c351fd3c60ef4e28c1a50612761b13e#a159cfa75abc87777645b7c926151130 - Download (conancenter)
    pkgconf/2.2.0#4ac315b50ef734072b00ff3aacbf52bf:8b5cf1367c351fd3c60ef4e28c1a50612761b13e#480ad4ea88438b696114abd9c045863a - Download (conancenter)
    zlib/1.2.13#9df41c65e2c2b6ef47633dc32e0b699a:be7ccd6109b8a8f9da81fd00ee143a1f5bbd5bbf - Build

======== Installing packages ========

-------- Downloading 16 packages --------
cmake/3.31.8: Retrieving package 9e5323c65b94ae38c3c733fe12637776db0119a5 from remote 'conancenter' 
cmake/3.31.8: Downloading 77.5MB conan_package.tgz
cmake/3.31.8: Decompressing 77.5MB conan_package.tgz
cmake/3.31.8: Package installed 9e5323c65b94ae38c3c733fe12637776db0119a5
cmake/3.31.8: Downloaded package revision dc85dedb7de60a1e60e0d9b131020248
cmake/4.1.1: Retrieving package 9e5323c65b94ae38c3c733fe12637776db0119a5 from remote 'conancenter' 
cmake/4.1.1: Downloading 80.7MB conan_package.tgz
cmake/4.1.1: Decompressing 80.7MB conan_package.tgz
cmake/4.1.1: Package installed 9e5323c65b94ae38c3c733fe12637776db0119a5
cmake/4.1.1: Downloaded package revision 5cc7773db11469b6c7f5db81b75593c1
gnu-config/cci.20210814: Retrieving package da39a3ee5e6b4b0d3255bfef95601890afd80709 from remote 'conancenter' 
gnu-config/cci.20210814: Package installed da39a3ee5e6b4b0d3255bfef95601890afd80709
gnu-config/cci.20210814: Downloaded package revision 22618e30bd9e326eb95e824dc90cc860
gperf/3.1: Retrieving package 9ac8640923e5284645f8852ef8ba335654f4020e from remote 'conancenter' 
gperf/3.1: Package installed 9ac8640923e5284645f8852ef8ba335654f4020e
gperf/3.1: Downloaded package revision 858dd07257747c43e443e77c5db10e5a
lzma_sdk/9.20: Retrieving package 82339cc4d6db7990c1830d274cd12e7c91ab18a1 from remote 'conancenter' 
lzma_sdk/9.20: Package installed 82339cc4d6db7990c1830d274cd12e7c91ab18a1
lzma_sdk/9.20: Downloaded package revision 36bd52c4dd7a7f4b470cac43f25bd621
m4/1.4.19: Retrieving package 9ac8640923e5284645f8852ef8ba335654f4020e from remote 'conancenter' 
m4/1.4.19: Package installed 9ac8640923e5284645f8852ef8ba335654f4020e
m4/1.4.19: Downloaded package revision 0b930e2019f5dfbf0c5e4ed8b8be9bf1
nasm/2.15.05: Retrieving package 9ac8640923e5284645f8852ef8ba335654f4020e from remote 'conancenter' 
nasm/2.15.05: Package installed 9ac8640923e5284645f8852ef8ba335654f4020e
nasm/2.15.05: Downloaded package revision 5b06c4e96fdf7a4baba8ad949a2ed29f
ninja/1.13.1: Retrieving package 9ac8640923e5284645f8852ef8ba335654f4020e from remote 'conancenter' 
ninja/1.13.1: Package installed 9ac8640923e5284645f8852ef8ba335654f4020e
ninja/1.13.1: Downloaded package revision f211295e2983ea2b0e62fc97e0093251
pkgconf/2.0.3: Retrieving package 8b5cf1367c351fd3c60ef4e28c1a50612761b13e from remote 'conancenter' 
pkgconf/2.0.3: Package installed 8b5cf1367c351fd3c60ef4e28c1a50612761b13e
pkgconf/2.0.3: Downloaded package revision a3e961a648771db40b115af998e14ac4
pkgconf/2.1.0: Retrieving package 8b5cf1367c351fd3c60ef4e28c1a50612761b13e from remote 'conancenter' 
pkgconf/2.1.0: Package installed 8b5cf1367c351fd3c60ef4e28c1a50612761b13e
pkgconf/2.1.0: Downloaded package revision a159cfa75abc87777645b7c926151130
pkgconf/2.2.0: Retrieving package 8b5cf1367c351fd3c60ef4e28c1a50612761b13e from remote 'conancenter' 
pkgconf/2.2.0: Package installed 8b5cf1367c351fd3c60ef4e28c1a50612761b13e
pkgconf/2.2.0: Downloaded package revision 480ad4ea88438b696114abd9c045863a
autoconf/2.71: Retrieving package 9e5323c65b94ae38c3c733fe12637776db0119a5 from remote 'conancenter' 
autoconf/2.71: Package installed 9e5323c65b94ae38c3c733fe12637776db0119a5
autoconf/2.71: Downloaded package revision c693b1ceb6f260246f50440ff7544168
meson/1.2.1: Retrieving package da39a3ee5e6b4b0d3255bfef95601890afd80709 from remote 'conancenter' 
meson/1.2.1: Package installed da39a3ee5e6b4b0d3255bfef95601890afd80709
meson/1.2.1: Downloaded package revision f7ef1db356d1746b23003b7924eb2128
meson/1.4.0: Retrieving package da39a3ee5e6b4b0d3255bfef95601890afd80709 from remote 'conancenter' 
meson/1.4.0: Package installed da39a3ee5e6b4b0d3255bfef95601890afd80709
meson/1.4.0: Downloaded package revision 91b870cdcf4edb1a302a2ef7a0514791
meson/1.8.2: Retrieving package da39a3ee5e6b4b0d3255bfef95601890afd80709 from remote 'conancenter' 
meson/1.8.2: Package installed da39a3ee5e6b4b0d3255bfef95601890afd80709
meson/1.8.2: Downloaded package revision 4bc04da65e30aac060f4c7aedf23be78
automake/1.16.5: Retrieving package 9e5323c65b94ae38c3c733fe12637776db0119a5 from remote 'conancenter' 
automake/1.16.5: Package installed 9e5323c65b94ae38c3c733fe12637776db0119a5
automake/1.16.5: Downloaded package revision 41880d59cb9b7157938fd9ee94827e51
brotli/1.1.0: Sources downloaded from 'conancenter'
brotli/1.1.0: Calling source() in /Users/runner/.conan2/p/brotlab92f392db1eb/s/src
brotli/1.1.0: Uncompressing v1.1.0.tar.gz to .


...


fftw/3.3.10: Sources downloaded from 'conancenter'
fftw/3.3.10: Calling source() in /Users/runner/.conan2/p/fftwde7b492d90aea/s/src
fftw/3.3.10: Uncompressing fftw-3.3.10.tar.gz to .

-------- Installing package fftw/3.3.10 (7 of 67) --------
fftw/3.3.10: Building from source
fftw/3.3.10: Package fftw/3.3.10:4dc90630c36eda1f80f6d3bc636c419a05107dc1
fftw/3.3.10: settings: os=Macos arch=x86_64 compiler=apple-clang compiler.version=15 build_type=Debug
fftw/3.3.10: options: fPIC=True openmp=False precision_double=True precision_longdouble=True precision_quad=False precision_single=True shared=False simd=False threads=False
fftw/3.3.10: Copying sources to build folder
fftw/3.3.10: Building your package in /Users/runner/.conan2/p/b/fftwa4ca9971f3084/b
fftw/3.3.10: Calling generate()
fftw/3.3.10: Generators folder: /Users/runner/.conan2/p/b/fftwa4ca9971f3084/b/build/Debug/generators
fftw/3.3.10: CMakeToolchain generated: conan_toolchain.cmake
fftw/3.3.10: CMakeToolchain generated: /Users/runner/.conan2/p/b/fftwa4ca9971f3084/b/build/Debug/generators/CMakePresets.json
fftw/3.3.10: CMakeToolchain generated: /Users/runner/.conan2/p/b/fftwa4ca9971f3084/b/src/CMakeUserPresets.json
fftw/3.3.10: Generating aggregated env files
fftw/3.3.10: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
fftw/3.3.10: Calling build()
fftw/3.3.10: apply_conandata_patches(): No patches defined in conandata
fftw/3.3.10: Running CMake.configure()
fftw/3.3.10: RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/Users/runner/.conan2/p/b/fftwa4ca9971f3084/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Debug" -DENABLE_FLOAT="ON" -DENABLE_LONG_DOUBLE="OFF" -DENABLE_QUAD_PRECISION="OFF" "/Users/runner/.conan2/p/b/fftwa4ca9971f3084/b/src"
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!

fftw/3.3.10: ERROR: 
Package '4dc90630c36eda1f80f6d3bc636c419a05107dc1' build failed
fftw/3.3.10: WARN: Build folder /Users/runner/.conan2/p/b/fftwa4ca9971f3084/b/build/Debug
ERROR: fftw/3.3.10: Error in build() method, line 111
	cmake.configure(variables=variables)
	ConanException: Error 1 while executing
Error: Process completed with exit code 1.
```

</details>

--- a/recipes/fftw/all/conanfile.py
+++ b/recipes/fftw/all/conanfile.py
@@ -90,6 +90,7 @@ class FFTWConan(ConanFile):
         tc.variables["ENABLE_SSE2"] = self.options.simd == "sse2"
         tc.variables["ENABLE_AVX"] = self.options.simd == "avx"
         tc.variables["ENABLE_AVX2"] = self.options.simd == "avx2"
+        tc.cache_variables["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5" # CMake 4 support
         tc.generate()
 
     @property

